gradle/libs.versions.toml

```diff
@@ -50,7 +50,7 @@ protobuf = "4.28.2"
 protobufPlugin = "0.9.4"
 retrofit = "2.11.0"
 retrofitKotlinxSerializationJson = "1.0.0"
-robolectric = "4.11.1"
+robolectric = "4.12.2"
```

Comment:
What about going to Robolectric 4.14.1?

Reply:
Good spot. For some reason that didn't show up in Android Studio.